packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -43,25 +49,36 @@ export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
             border: '1px solid WindowText'
           },
           ':hover': {
-            borderColor: palette.neutralDark,
+            borderColor: semanticColors.inputBorderHovered,
             selectors: {
               [HighContrastSelector]: {
                 borderColor: 'Highlight'
               }
             }
           },
           [`:hover .${classNames.iconContainer}`]: {
-            color: palette.themeDark
+            color: semanticColors.inputIconHovered
           }
         }
       },
+      !hasFocus &&
+        hasInput && {
+          selectors: {
+            [`:hover .${classNames.iconContainer}`]: {
```

Comment:
@phkuo just a heads up, this is an example of referring to other parts! `iconContainer` is a generated classname that will be stuck on the correct element, and will never mutate. That way we don't have to recalculaate this selector.

@maxwellred nice work :)

Reply:
@dzearing You bet! Although I have to give the credit to @Vitalius1 for making that great catch in code review on the PR I cherry-picked these changes from.